lib/prediction/predictor.ts

```diff
@@ -52,92 +49,61 @@ interface Example {
     reject(err : Error) : void;
```

Comment:
```
const MINIBATCH_SIZE = 30;
const MAX_LATENCY = 50; //
````
Maybe make these two into parameters? Could be useful when we starting tuning GPU serving.

Reply:
Good idea